python-sdk/nuscenes/eval/detection/utils.py

```diff
@@ -214,6 +214,6 @@ def cummean(x: np.array) -> np.array:
         return np.ones(len(x))  # If all errors are NaN set to error to 1 for all operating points.
     else:
         # Accumulate in a nan-aware manner.
-        sum_vals = np.nancumsum(x)  # Cumulative sum ignoring nans.
+        sum_vals = np.nancumsum(x.astype(float))  # Cumulative sum ignoring nans.
```

Comment:
Note i did this because  `np.divide` will throw an error for np.array with single value that are `int`s. But if we feel that there won't be a chance that such input would occur then I don't mind removing it.

Reply:
Fine with me to keep. Thanks.